executor/set_test.go

```diff
@@ -325,6 +325,8 @@ func (s *testSuite2) TestSetVar(c *C) {
 	tk.MustQuery("select @@global.tx_isolation").Check(testkit.Rows("READ-UNCOMMITTED"))
 	tk.MustQuery("select @@global.transaction_isolation").Check(testkit.Rows("READ-UNCOMMITTED"))
 
+	tk.MustExec("SET GLOBAL transaction_isolation='REPEATABLE-READ'")
```

Comment:
What's this test case used for?

Reply:
we should reset tx_isolation back to rr before reset tidb_skip_isolation_level_check.

if we keep global tx_isolation in rc and reset tidb_skip_isolation_level_check to 0, when new session start and retry load global config into the session will meet `UnsupportedIsolationLevel` error.

Reply:
add some comments but maybe we should cascade reset tx_isolation to rr when tidb_skip_isolation_level_check be set to 0? :thinking: